pkg/cmd/synonyms/import/import.go

```diff
@@ -167,7 +188,7 @@ func runImportCmd(opts *ImportOptions) error {
 
 	if count > 0 {
 		totalCount += count
-		if _, err := indice.SaveSynonyms(batch); err != nil {
+		if _, err := indice.SaveSynonyms(batch, batchOptions...); err != nil {
```

Comment:
Not related to the PR here but I don't understand why in Go, we use `...` when passing a slice as an argument to a variadic function: the fact that `batchOptions` here is already a slice isn't explicit enough?

Reply:
Yes, you need to "unpack" the slice first. Otherwise, the `batchOptions` would be considered as a single arg (of the wrong type).